src/docs/Components/TextInput/index.mdx

```diff
@@ -30,3 +36,7 @@ Use a left icon to inform about a specific information type or formatting you ex
 <HintItem>
   Always use a label associated with an input, even using placeholders
 </HintItem>
+
+<HintItem dont>
```

Comment:
Can we add a "don't" to specity this: Don't use a medium input with a small button.
What do you think?

Reply:
fixed